packages/errors/src/messages.ts

```diff
@@ -39,6 +40,7 @@ export const SolanaErrorMessages: Readonly<{
         'The $argumentLabel argument to the `$methodName` RPC method$optionalPathLabel was ' +
         '`$value`. This number is unsafe for use with the Solana JSON-RPC because it exceeds ' +
         '`Number.MAX_SAFE_INTEGER`.',
+    [SOLANA_ERROR__SUBTLE_CRYPTO]: 'SubtleCrypto error: $reason',
```

Comment:
I think this error is too vague and we should create multiple granular errors instead such as:

- `SOLANA__ERROR_SUBTLE_CRYPTO_MISSING`
- `SOLANA__ERROR_SUBTLE_CRYPTO_ED25519_ALGORITHM_MISSING`
- `SOLANA__ERROR_SUBTLE_CRYPTO_VERIFY_FUNCTION_MISSING`
- ...

Which would remove the need for the `reason` string in this case.

I think in general, if you need `reason` it's a smell.

Reply:
Discussion: https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2187/refactor-experimental-errors-addresses-package#discussion-IC_kwDOCLAddc51Bk0K